### ISSUE TITLE
Crash on Import in Blender 3.6

### ISSUE THREAD

**Issue by FeuerTiger86** [OGRECave/blender2ogre#204]
#### System Information
- Ogre Version: v14.2.1
- Operating System / Platform: Windows 11
- Blender Version: 3.6.8 (Portable)

#### Detailed description
Importing a .mesh file makes Blender crash on version 3.6.8, without error message. It works on 2.93.18 and 3.3.16. The mesh was exported with this add-on.

#### blender2ogre.log
```
2024-03-06 11:21:28 [ INFO] Target_path: C:\Users\pmuell10\Documents\Blender
2024-03-06 11:21:28 [ INFO] Target_file_name: Suzanne.mesh
2024-03-06 11:21:28 [ INFO] * Loading mesh from: C:\Users\pmuell10\Documents\Blender\Suzanne.mesh
2024-03-06 11:21:28 [ INFO] * Parsing file: C:\Users\pmuell10\Documents\Blender\Suzanne.mesh.xml ...
2024-03-06 11:21:28 [ INFO] - Done at 0.15 seconds
2024-03-06 11:21:28 [ INFO] * Collecting mesh data...
2024-03-06 11:21:29 [ INFO] * Collecting pose data...
2024-03-06 11:21:29 [ INFO] + Creating mesh: Suzanne
```

[blender_debug_output.txt](https://github.com/OGRECave/blender2ogre/files/14508816/blender_debug_output.txt)

**Comment by sercero** [comment]
Hello, are you using the latest version from `master`?

Because I think that issue has been solved already.

**Comment by FeuerTiger86** [comment]
Oh right, I haven't, I used the 0.8.4 release. I will try out the master version as soon as I can. Thank you!

**Comment by sercero** [comment]
@FeuerTiger86 have you tried it yet?

**Comment by FeuerTiger86** [comment]
Yes, I have. It didn't crash any more, but split normals were imported incorrectly. This is fixed in the new release, however.

**Comment by sercero** [comment]
Split normals are still imported incorrectly?

Sorry if I misunderstood.

**Comment by FeuerTiger86** [comment]
They are imported correctly with the new release version.
